pkg/apb/ext_creds_test.go

```diff
@@ -7,197 +7,23 @@ import (
 	ft "github.com/openshift/ansible-service-broker/pkg/fusortest"
 )
 
-func TestDecodeOutput(t *testing.T) {
-	output := []byte(`
```

Comment:
Because decodeOuput now it only does ```decodedjson, err := base64.StdEncoding.DecodeString(str)```.  The only thing I would be testing is if a base64 library works.